docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -220,6 +223,13 @@ The following table shows the Onvif functions tested for various Onvif cameras:
 * 'ⓘ' means there is additional details available. Click it to read more.
 * Empty cells means the function has not yet been tested.
 
+### Camera Documentation
+Use these links to access maufacturer documentation
+
+| Hikvision DFI6256TE | Tapo C200 | BOSCH DINION IP starlight 6000 HD | GeoVision GV-BX8700 | Hikvision DS-2DE2A404IW-DE3 |
+|---------------------|-----------|-----------------------------------|---------------------|---------------------|
+| [Hikvision DFI6256TE](https://www.authority.com.tw/product-detail-2707792.html) | [Tapo C200](https://www.tapo.com/us/product/smart-camera/tapo-c200/) | [BOSCH DINION IP starlight 6000 HD](https://commerce.boschsecurity.com/tw/en/DINION-IP-starlight-6000-HD/p/20827877387/)| [GeoVision GV-BX8700](https://www.geovision.com.tw/product/GV-BX8700-FD) | [Hikvision DS-2DE2A404IW-DE3](https://us.hikvision.com/en/products/cameras/network-ptz-camera/value-series/ir/outdoor/4-mp-4x-ir-outdoor-network-ptz-camera-ds) |
+
```

Comment:
The contents of this table are redundant with the headers. Suggest restructuring to just a bulleted list
```suggestion
- [Hikvision DFI6256TE](https://www.authority.com.tw/product-detail-2707792.html)
- [Tapo C200](https://www.tapo.com/us/product/smart-camera/tapo-c200/) 
- [BOSCH DINION IP starlight 6000 HD](https://commerce.boschsecurity.com/tw/en/DINION-IP-starlight-6000-HD/p/20827877387/)
- [GeoVision GV-BX8700](https://www.geovision.com.tw/product/GV-BX8700-FD) 
- [Hikvision DS-2DE2A404IW-DE3](https://us.hikvision.com/en/products/cameras/network-ptz-camera/value-series/ir/outdoor/4-mp-4x-ir-outdoor-network-ptz-camera-ds) |

```